ReactWindows/ReactNative.Shared/Views/TextInput/ReactPasswordBoxShadowNode.cs

```diff
@@ -260,7 +271,7 @@ private static string GetDefaultPasswordChar()
             if (s_passwordChar == null)
             {
                 var passwordBox = new PasswordBox();
-                s_passwordChar = passwordBox.PasswordChar;
+                s_passwordChar = passwordBox.PasswordChar.ToString();
```

Comment:
If we have #ifs in here anyway, can we if this so we don't have the unnecessary call to ToString() for UWP?



---
In reply to: [89251699](https://github.com/ReactWindows/react-native-windows/pull/900#discussion_r89251699) [](ancestors = 89251699)